src/command/render/render.ts

```diff
@@ -199,7 +199,11 @@ export async function renderPandoc(
   }
 
   // run pandoc conversion (exit on failure)
-  const pandocResult = await runPandoc(pandocOptions, executeResult.filters);
+  const pandocResult = await runPandoc(
+    pandocOptions,
+    executeResult.filters,
+    context.engine.name,
```

Comment:
Everything else looks good, but I think we can improve this change here. Notice that pandocOptions already has `executionEngine` (see src/command/render/types.ts). We should use that one, right?

Reply:
> Notice that pandocOptions already has executionEngine (see src/command/render/types.ts). We should use that one, right?

😓 I looked for to see if it was already in, and missed this. This is definitely better to use it.